### PR TITLE
chore: Added Few more sections to contributing.md

### PR DESCRIPTION
**Pull request by devcer**
## Changes proposed in this pull request
Added `Reporting issues`, `Submitting Pull Requests` and `Review Process` sections to contributing.md

- 

## Context
These additional sections will guide the new contributors from picking up the issue to successfully submitting their PR.

## Checklist

- [ ] Related issues linked using `fixes #number`
- [ ] Tests added/updated
- [x] Documentation added
- [ ] Make sure that all checks pass
- [ ] Postman collection updated

### CODE REVIEW

File: .github/contributing.md [hunk 1]
```diff
@@ -163,3 +164,32 @@ each package to suit the package's requirements.
...
+- Information about your environment (e.g., operating system, browser, version).
+- Any relevant screenshots or error messages.
+
+## Submitting Pull Requests
+
+1. [Fork](https://github.com/interledger/rafiki) the repository.
+2. Create a new branch from `main`.
+3. Make your changes and commit them.
+4. Create a pull request (PR) to `main`.
+5. Ensure your PR includes a clear title and description.
```
Comment (AlexLakatos) [inline .github/contributing.md:183]: we need to add a line here that talks about our PR title requirements for conventional commits: https://www.conventionalcommits.org/en/v1.0.0/

Comment (devcer) [reply .github/contributing.md]: Added.

File: .github/contributing.md [hunk 2]
```diff
@@ -1,51 +1,55 @@
...
 
-### Table of Contents <!-- omit in toc -->
+## Table of Contents <!-- omit in toc -->
 
 - [Types of contributions](#types-of-contributions)
   - [:mega: Discussions](#mega-discussions)
   - [:beetle: Issues](#beetle-issues)
   - [:hammer_and_wrench: Pull requests](#hammer_and_wrench-pull-requests)
 - [Working in the rafiki repository](#working-in-the-rafiki-repository)
-  - [Workspaces](#workspaces)
```
Comment (devcer) [inline .github/contributing.md:18]: This looks like a redundant item so removed it.

File: .github/contributing.md [hunk 3]
```diff
@@ -1,51 +1,55 @@
...
 ### :hammer_and_wrench: Pull requests
+
 Feel free to fork and create a pull request on changes you think you can contribute.
 
 The team will review your pull request as soon as possible.
 
 ### :books: Documentation
-We started to maintain public-facing documentation for Rafiki on [rafiki.dev](https://github.com/interledger/rafiki.dev). The project is new, and available documentation there is scarce. We'd love help to fill that out. A list of issues being tracked across the Interledger ecosystem (including rafiki) is maintained in the [Documentation project](https://github.com/orgs/interledger/projects/5/views/1).
 
+We started to maintain public-facing documentation for Rafiki on [rafiki.dev](https://github.com/interledger/rafiki.dev). The project is new, and available documentation there is scarce. We'd love help to fill that out. A list of issues being tracked across the Interledger ecosystem (including rafiki) is maintained in the [Documentation project](https://github.com/orgs/interledger/projects/5/views/1).
```
Comment (AlexLakatos) [inline .github/contributing.md:52]: That's not where documentation lives now. We've moved that inside the rafiki repository: https://github.com/interledger/rafiki/tree/main/packages/documentation

Comment (devcer) [reply .github/contributing.md]: Done.

File: .github/contributing.md [hunk 4]
```diff
@@ -1,51 +1,55 @@
...
 ### :hammer_and_wrench: Pull requests
+
 Feel free to fork and create a pull request on changes you think you can contribute.
 
 The team will review your pull request as soon as possible.
 
 ### :books: Documentation
-We started to maintain public-facing documentation for Rafiki on [rafiki.dev](https://github.com/interledger/rafiki.dev). The project is new, and available documentation there is scarce. We'd love help to fill that out. A list of issues being tracked across the Interledger ecosystem (including rafiki) is maintained in the [Documentation project](https://github.com/orgs/interledger/projects/5/views/1).
 
+We started to maintain public-facing documentation for Rafiki on [rafiki.dev](https://github.com/interledger/rafiki.dev). The project is new, and available documentation there is scarce. We'd love help to fill that out. A list of issues being tracked across the Interledger ecosystem (including rafiki) is maintained in the [Documentation project](https://github.com/interledger/rafiki/tree/main/packages/documentation).
```
Comment (AlexLakatos) [inline .github/contributing.md:52]:
```suggestion
We started to maintain public-facing documentation for Rafiki on [rafiki.dev](https://github.com/interledger/rafiki/tree/main/packages/documentation). The project is new, and available documentation there is scarce. We'd love help to fill that out. A list of issues being tracked across the Interledger ecosystem (including rafiki) is maintained in the [Documentation project](https://github.com/interledger/rafiki/tree/main/packages/documentation).
```

**Review by AlexLakatos — APPROVED** [review]

File: .github/contributing.md [hunk 5]
```diff
@@ -1,51 +1,55 @@
...
 ### :hammer_and_wrench: Pull requests
+
 Feel free to fork and create a pull request on changes you think you can contribute.
 
 The team will review your pull request as soon as possible.
 
 ### :books: Documentation
-We started to maintain public-facing documentation for Rafiki on [rafiki.dev](https://github.com/interledger/rafiki.dev). The project is new, and available documentation there is scarce. We'd love help to fill that out. A list of issues being tracked across the Interledger ecosystem (including rafiki) is maintained in the [Documentation project](https://github.com/orgs/interledger/projects/5/views/1).
 
+We started to maintain public-facing documentation for Rafiki on [rafiki.dev](https://github.com/interledger/rafiki/tree/main/packages/documentation). The project is new, and available documentation there is scarce. We'd love help to fill that out. A list of issues being tracked across the Interledger ecosystem (including rafiki) is maintained in the [Documentation project](https://github.com/interledger/rafiki/tree/main/packages/documentation).
```
Comment (AlexLakatos) [inline .github/contributing.md:52]:
```suggestion
We started to maintain public-facing documentation for Rafiki on [rafiki.dev](https://github.com/interledger/rafiki/tree/main/packages/documentation). The project is new, and available documentation there is scarce. We'd love help to fill that out. A [list of issues](https://github.com/interledger/rafiki/issues?q=is%3Aissue+is%3Aopen+label%3Adocumentation) is available.
```

**Review by AlexLakatos — APPROVED** [review]

**Pull request merged**